src/__tests__/to-have-prop.tsx

```diff
@@ -4,20 +4,20 @@ import { render } from '@testing-library/react-native';
 
 describe('.toHaveProp', () => {
   const { queryByTestId } = render(
-    <View accessibilityLabel={null} testID="view">
-      <Text allowFontScaling={false} testID="text" editable={false}>
+    <View style={null} testID="view">
```

Comment:
Is there a reason for swapping these props here?

Reply:
View doesn't have `accessibilityLabel` on the type definition and as now the file runs on typescript it complains. I changed it for style as i think still respect the intention of the original test.

Reply:
That's weird, according to RN types def it should have it:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/482d94eee7b27c478034e188bbeb64e2f995bbd8/types/react-native/index.d.ts#L2531

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/482d94eee7b27c478034e188bbeb64e2f995bbd8/types/react-native/index.d.ts#L2309 the type for it is `string | undefined` so it complains about null. i can change it to `accessibilityLabel={undefined}` or nothing and check  `accessibilityLabel ` later but wanted to respect the original intention of the test